### PR TITLE
Backport 8232594: Make the output of the JFR command duration more user friendly

### PR DESCRIPTION
**Pull request by aamarsh**
[Original JBS issue](https://bugs.openjdk.org/browse/JDK-8232594)

[Original Commit](https://github.com/openjdk/jdk/commit/558aadf60818b30e57b2462c331c0a8810739670)

Clean backport, no changes to the code from the original backport. Minor white space changes.

Added jfr test `TestFormatDuration.java` is passing locally.

### CODE REVIEW

**Comment by karianna** [comment]
I won't comment on the correctness since I'm not qualified, but I would suggest putting in comments into the source code as per your Differences message from above.  It'll make it easier for you to debug in the future.

File: src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java [hunk 1]
```diff
@@ -57,9 +57,6 @@
  * This class is also used by {@link RecordedObject#toString()}
  */
 public final class PrettyWriter extends EventPrintWriter {
-    private static final Duration MILLSECOND = Duration.ofMillis(1);
```
Comment (aamarsh) [inline src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java:60]: The original PR changes the copyright date at the top of this file. I did not include this change.

Comment (brianjstafford) [reply src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java:60]: Hmm, I'm not sure what the coding convention in OpenJDK is for updating copyright dates. Can you please check on that? We may want to preserve the update from the original PR (or possibly even use 2022).

Comment (karianna) [reply src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java:60]: I'd recommend pulling in the new date (that's the unspoken convention)

Comment (aamarsh) [reply src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java:60]: I copied the original backport with the copyright. Now this is an entirely clean backport.

**Comment by aamarsh** [comment]
@karianna and @brianjstafford do you know if it is still considered a "clean backport" if there were merge conflicts where extra spaces and formatting needed to be fixed because all the code around the changes is very different from tip to 11? All the code that has been added is the exact same but I think there are one or two newlines and the process wasn't necessarily clean with merge conflicts...

**Comment by karianna** [comment]
You can say clean backport (modulo white space formatting)

**Comment by kthatipally** [comment]
Closing the draft PR as the corresponding changes are released as a part of [jdk-11.0.18+10](https://github.com/openjdk/jdk11u/releases/tag/jdk-11.0.18%2B10).

Corresponding ms-patches commits: https://github.com/microsoft/openjdk-jdk11u/commit/0fc1d17a6a1ba4f7630bf2865e10d44c945a28a8, https://github.com/microsoft/openjdk-jdk11u/commit/7ea23efa610bb12757389cac5a7e7e64c1e846a0

**Pull request closed without merging**